armi/reactor/converters/geometryConverters.py

```diff
@@ -789,6 +780,105 @@ def _createRadialThetaZone(
         self.convReactor.core.add(newAssembly)
 
         return outerDiameter
+    
+    def _addNewComponents(self, newBlock, innerDiameter, outerDiameter, 
+                          lowerAxialZ, upperAxialZ, lowerTheta, upperTheta):
+        """
+        Add a set of components to the new homogenized block that was created.
+        
```

Comment:
@ntouran I think this method will have to be re-written soon, as Components can now have sub-components.

Thoughts?

Reply:
This class just wouldn't be able to support that. Maybe we need a namespace for classes that will not be that general. Lots of code in the framework assumes a reactor -> core -> assembly -> block -> component -> material relationship and it's going to difficult to generalize or take lots of work to generalize. Not sure how we would like to handle generic composite trees going forward